deployment/node-feature-discovery/templates/nfd-worker-conf.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "node-feature-discovery.fullname" . }}
```

Comment:
Hmm, this must be aligned with the mount in nfd-worker. A separate configuration parameter in values.yaml e.g. `worker.configmapName`(?)

Reply:
Good catch, will fix it asap